### PR TITLE
Use ableton.pkg_mgr_path paths

### PR DESCRIPTION
**Pull request by nre-ableton**
- Introduce dependency on ableton.pkg_mgr_path
- Use ableton.pkg_mgr_path paths
- Version 0.1.1

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ devToolsProject.run(
   setup: { data ->
     Object venv = virtualenv.create('python3.8')
     venv.run('pip install -r requirements-dev.txt')
+    venv.run('ansible-galaxy install -f -r requirements.yml')
```
Comment (ala-ableton) [inline Jenkinsfile:10]:
```suggestion
    venv.run('ansible-galaxy install --force --role-file requirements.yml')
```

**Review by ala-ableton — COMMENTED** [review]
Please rebase/squash and I'll have a final look, thanks!

**Comment by mst-ableton** [comment]
@ablbot autosquash

**Review by ala-ableton — APPROVED** [review]

**Review by mst-ableton — APPROVED** [review]
nice! lgtm

**Pull request merged**